keycloak-init-container/extensions.sh

```diff
@@ -15,7 +15,18 @@ download_extension() {
     fi
 
     echo "Downloading extension from $EXTENSION_URL"
-    local FILENAME="$(curl --verbose --location --remote-name --remote-header-name --write-out '%{filename_effective}' --silent "$EXTENSION_URL" 2> /tmp/headers)"
+    local CURL_COMMAND="$(curl --verbose --location  --remote-name --remote-header-name --write-out "%{response_code} %{filename_effective}"  --silent "$EXTENSION_URL" 2> /tmp/headers)"
+
+    local STATUS_CODE=${CURL_COMMAND:0:3}
+
+    if [ $STATUS_CODE -eq "200" ]; then
+        echo "Extension downloaded successfully"
+    else
+        echo "Can not download the extension"
```

Comment:
Could we return, what extension exactly was causing problems?

Reply:
Thank you for your comment @slaskawi, sure, I'll update the PR and try to figure out the error codes, for now, any unsuccessful return is 000 while a successful is 200.

Reply:
Also, watch out for curl errors... e.g. what happens if network is unavailable etc. 

The aim here is to give an administrator enough information to figure out whether it's local Kubernetes problems or maybe the target web page misbehaves (e.g. Github returning 503 etc).

Reply:
Thanks @slaskawi , I'll give it a go!